lib/redmine/views/my_page/block.rb

```diff
@@ -34,23 +34,22 @@ module Block
         def self.additional_blocks
           # look at the gemspecs of all plugins trying to find views in a /my/blocks subdirectory
           @@additional_blocks ||= Dir.glob(
-            Plugin.registered_plugins.map do |plugin_id, _|
+            Plugin.registered_plugins.map { |plugin_id, _|
               gem_name = plugin_id.to_s.gsub('openproject_', 'openproject-') if plugin_id.to_s.starts_with?('openproject_')
               gem_spec = Gem.loaded_specs[gem_name]
               if gem_spec.nil?
-                error = 'No Gemspec found for plugin: ' + plugin_id.to_s \
-                + ', expected gem name to match the plugin name but starting with openproject-'
+                error = 'No Gemspec found for plugin: ' + plugin_id.to_s + ', expected gem name to match the plugin name but starting with openproject-'
```

Comment:
Line is too long. [152/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.